guides/auth0.md

```diff
@@ -11,10 +11,16 @@ There are two connections configured currently, Google and Microsoft. The former
 
 Currently the only user data we keep is their name and email address, both of which are encrypted before being saved in the DB.
 
-The Universal Login screen can be configured directly in the moj-forms tenant dashboard. It is possible to create custom HTML and JS.
+The Universal Login has been configured within the Auth0 dashboard. Any changes should be done there directly. The username and password field are required only in editor-test as they are used by the fb-acceptance-tests user.
 
 There are two editor apps, one for Live and one for Test. Aside from the obvious differences between callback URLs, the Test app also has the username and password fields active in order for the acceptance tests to be able to log in.
 
+The tenant environment is usually set to "development" and needs to have a manual request through the Auth0 service desk to set it to "production". The best situation is to have separate tenants per environment however it so long to get the connections set up that to go back and request IDs and private keys just is not worth the time and effort.
```

Comment:
just missing the word 'took'  in '...tenants per environment however it so long...'